ui/lib/components/MultiSelect/index.tsx

```diff
@@ -47,9 +47,7 @@ export interface IMultiSelectProps<T>
   columnTitle?: string
 }
 
-export default function MultiSelect<T extends IItem>(
-  props: IMultiSelectProps<T>
-) {
+function MultiSelect<T extends IItem>(props: IMultiSelectProps<T>) {
```

Comment:
Changing to this writing style to pass the storybook compilation. Ref: https://github.com/storybookjs/storybook/issues/11419#issuecomment-658969643